.github/ISSUE_TEMPLATE/documentation_isse.yml

```diff
@@ -0,0 +1,21 @@
+name: 📚 Documentation Issue
```

Comment:
We have a lot of documentation updates filed as bugs so added a separate template here for those